app/scripts/lib/app-start.js

```diff
@@ -238,9 +237,6 @@ define(function (require, exports, module) {
         // If in an iframe for sync, the origin is checked against
         // a pre-defined set of origins sent from the server.
         return this._config.allowedParentOrigins;
-      } else if (this._isOAuth()) {
-        // If in oauth, the relier has the allowed parent origin.
-        return [this._relier.get('origin')];
```

Comment:
Is this the only place we use the 'origin' property from the relier?  A grep suggests it might be, and if so, we should also remove it from the model.